game_server_launcher/launcher.py

```diff
@@ -295,6 +295,7 @@ def handle_add_player_message(self, msg):
 
     def handle_remove_player_message(self, msg):
         if msg.ip:
+            display_name = msg.display_name
```

Comment:
I think you intended to print it on the line below and this assignment can be removed, correct?

Reply:
actually this should not be here at all. We only need it on join